overview.md

```diff
@@ -1,37 +1,38 @@
-Jupyter transitioned from a [Benevolent Dictator For Life (BDFL) + Steering Council](governance.md) governance model to this current governance model in December 2022.
+Jupyter transitioned from a [Benevolent Dictator For Life (BDFL) + Steering Council](archive/governance.md) governance model to this current governance model in December 2022.
 
 This document provides a brief informational summary of the Project Jupyter governance model. In case of any substantive discrepancy with the official descriptions of each body, the underlying governance documents should be considered as the source of truth, and we will update this overview as needed.
 :::
 
-Jupyter’s governance model is anchored on two bodies that complement each other:
+Jupyter’s governance model is anchored on three bodies that complement each other:
 
 1. The [**Executive Council (EC)**](executive_council.md) is ultimately responsible for all dimensions of the Project (including, but not limited to, software, legal, financial, community, operations, inclusion and diversity, etc.). The members of the EC actively work to carry out the Project's mission in accordance with its values and to support operations through delegation to the Software Steering Council (SSC), Software Subprojects, Standing Committees, and Working Groups. These other bodies will report to the EC, and the EC is expected to support, oversee, manage, and ensure the success of operations across Jupyter. For more detail, see the [Executive Council document](executive_council.md).
 2. The [**Software Steering Council (SSC)**](software_steering_council.md) has jurisdiction over software-related decisions across Project Jupyter, with a primary focus on coordination across projects and decisions that have impact across many Jupyter Subprojects. It is also a mechanism for representatives of each project to share information and expertise. Technical decisions and processes where the SSC isn't explicitly involved are automatically delegated to the individual projects to manage their day-to-day activities, create new repositories in their orgs, etc., with independence and autonomy. For more details, see the [Software Steering Council document](software_steering_council.md).
+3. The [**Jupyter Foundation**](./jupyter_foundation.md) is a directed fund of the Linux Foundation 501(c)(6) that exists to provide resources and strategic counsel to Project Jupyter. The Executive Council serves on the Jupyter Foundation Governing Board. For more details, see the [Jupyter Foundation document](./jupyter_foundation.md).
 
 Additionally, the Executive Council (EC) receives input from a Community Advisory Panel. This panel advises the EC with perspectives and connections that may reach beyond the active Jupyter community.
 
 ## Other major components of the organization
 
 In addition to these three bodies, the following are other major parts of the Project related to governance.
 
-### Distinguished Contributors
+### Software Subprojects
```

Comment:
I've opened #228 to handle this re-ordering. I will update this PR so that we don't do any re-ordering here and can keep this PR focused on the LF updates.